graphrag/index/cli.py

```diff
@@ -113,7 +113,7 @@ def index_cli(
     """Run the pipeline with the given config."""
     progress_reporter = load_progress_reporter(reporter or "rich")
     info, error, success = _logger(progress_reporter)
-    run_id = resume or time.strftime("%Y%m%d-%H%M%S")
+    run_id = resume or update_index_id or time.strftime("%Y%m%d-%H%M%S")
```

Comment:
Would be cool to add docs what happens when `resume` and `update_index_id` or if that's a mistake.

Reply:
Added an exception hehe